test/multistream.test.js

```diff
@@ -29,11 +29,11 @@ test('sends to multiple streams using string levels', function (t) {
   log.info('info stream')
   log.debug('debug stream')
   log.fatal('fatal stream')
-  t.equal(messageCount, 9)
-  t.end()
+  t.assert.strictEqual(messageCount, 9)
+  end()
```

Comment:
I don't think we need `end()` in these tests. With `tap`, we needed `t.end` because we didn't set forth a plan or return a promise. It looks to me like (at least most of) the tests in this suite are synchronous. `node:test`, at its most basic, only requires that a test not throw an error and the code of the test be fully synchronous.

Reply:
I left this comment on the first synchronous test, but there are still many others. Please update them as well, @Puppo.

Reply:
@jsumners done! I removed `end` where possible.